server/app/models/host_node.rb

```diff
@@ -68,7 +71,6 @@ def to_path
   # @param [Hash] attrs
   def attributes_from_docker(attrs)
     self.attributes = {
-      node_id: attrs['ID'],
```

Comment:
Considering re-adding this as some new `docker_id` field, but I'm not sure if it's needed.